pydarn/exceptions/rtp_exceptions.py

```diff
@@ -0,0 +1,92 @@
+# Authors: Marina Schmidt
+import logging
+import datetime
+pydarn_log = logging.getLogger('pydarn')
```

Comment:
Recommend adding an empty line before the `pydarn_log` definition to conform with the style of the other files.

Reply:
Here and a few other places.